atlp-rwanda/stackup2-barefoot-backend
# #171380743 Deploy this app to heroku

#### What does this PR ​ do​ ?
It deploys this Application to heroku

#### Description of Task to be completed?
* Add *Procfile* file which contains the web process to start when this app is deployed to heroku
* Add *app.json* file to tell heroku how to create *REVIEW APPS* whenever a new PR is raised
* Add a sample route in *index.js* file to test it the deployment worked

#### How should this be manually tested?
* Click [here]() if you see in your browser this message *{"message":"welcome"}* then the deployment is successful or scroll dow to this PR's deployment history, and click on `view deployment`

#### Any background context you want to provide?
To know how to use this deployment please read this [README](https://github.com/Stackup-Rwanda/stackup2-barefoot-backend/blob/ch-deploy-base-app-to-heroku-171380743/README.md) file on *Deployments* section

#### What are the relevant pivotal tracker stories?
[#171380743](https://www.pivotaltracker.com/n/projects/2435482)
#### Screenshots (​ if​ appropriate)
N/A
#### Questions:
N/A
